test/integration/test_upload_configuration_options.py

```diff
@@ -64,6 +67,24 @@ def test(self):
         # the newer API decorator that handles those details.
         assert create_response.status_code >= 400
 
+    @skip_without_datatype("velvet")
+    def test_composite_datatype(self):
+        payload = self.dataset_populator.upload_payload(
+            self.history_id,
+            "sequences content",
+            file_type="velvet",
+            extra_inputs={
+                "files_1|url_paste": "roadmaps content",
+                "files_1|type": "upload_dataset",
+                "files_2|url_paste": "file://%s/1.txt",
```

Comment:
The `%` doesn't seem right here.

Reply:
Yeah - good point. One second...

Reply:
Rebased with a fix for that - it doesn't change the test outcome really but it definitely was support to be an actual valid file URI. Thanks for the catch.